packages/pipcook-board-client/package.json

```diff
@@ -33,7 +33,7 @@
     "start": "ice-scripts dev",
     "build": "ice-scripts build",
     "lint": "npm run eslint && npm run stylelint",
-    "eslint": "eslint --cache --ext .js,.jsx ./",
+    "eslint": "eslint --cache --ext .js,.jsx ./ --fix",
```

Comment:
`--fix` should not be included the npm command, run the following if someone wants to fix:

```sh
$ npm run eslint -- --fix
```